resources/js/components/FormField.vue

```diff
@@ -124,6 +124,20 @@
 						return;
 					}
 
+                                        if (Array.isArray(dependency.value)) {
+                                            if (typeof this.dependencyValues[dependency.field] === 'object' && dependencyValue !== null) {
+                                                if (dependency.hasOwnProperty('value') && dependency.value.includes(dependencyValue.id)) {
```

Comment:
Based on how similar L129 conditional and L134 conditional are, can we find a way to combine those so that we aren't duplicating the conditional in 2 places? The only difference seems to be what we are seeing is included, dependencyValue.id vs just dependencyValue. All that duplication just means more code to manage and debug when something goes wrong